build/docker/intel-idxd-initcontainer.Dockerfile

```diff
@@ -12,38 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:unstable AS builder
+FROM debian:unstable-slim AS builder
 
-RUN apt-get update && apt-get install -y \
-            build-essential autoconf automake autotools-dev libtool \
-            pkgconf asciidoc xmlto uuid-dev libjson-c-dev libkmod-dev \
-            libudev-dev libkeyutils-dev curl
+RUN echo "deb-src http://deb.debian.org/debian unstable main" >> \
+        /etc/apt/sources.list.d/deb-src.list && \
+    apt update && apt install -y --no-install-recommends \
+        gcc make patch autoconf automake libtool pkg-config \
+        libjson-c-dev uuid-dev curl ca-certificates && \
+    cd /usr/local/src/ && apt --download-only source uuid libjson-c5 && cd /
```

Comment:
Go packages are copied under `/usr/local/share/package-sources`. Let's use that here too